.github/workflows/test.yml

```diff
@@ -391,7 +381,7 @@ jobs:
 
     - name: Runner Diagnostics
       if: always()
-      uses: ./.github/actions/diagnostics
+      uses: ./pr/.github/actions/diagnostics
```

Comment:
This happens after the PR branch checkout, so I think the path can stay as it was

Reply:
the previous action, a couple lines above is "Check out code into the Go module directory - from PR branch"?